cv_next/server/api/comments.ts

```diff
@@ -28,12 +28,13 @@ export async function addCommentToCv(
       return Err(
         addCommentToCv.name,
         undefined,
+        undefined,
         new Error("adding comment failed")
       );
     }
     return Ok.EMPTY;
   } catch (err) {
-    return Err(addCommentToCv.name, undefined, err as Error);
+    return Err(addCommentToCv.name, undefined, undefined, err as Error);
```

Comment:
Instead perhaps define Err with an object of errors (which can then be either `PostgresError`, `AuthError` etc)?